SLAM/EKFSLAM/ekf_slam.py

```diff
@@ -30,8 +30,8 @@ def ekf_slam(xEst, PEst, u, z):
     # Predict
     S = STATE_SIZE
     xEst[0:S] = motion_model(xEst[0:S], u)
-    G, Fx = jacob_motion(xEst[0:S], u)
-    PEst[0:S, 0:S] = G.T * PEst[0:S, 0:S] * G + Fx.T * Cx * Fx
+    G = jacob_motion(xEst[0:S], u)
+    PEst[0:S, 0:S] = G.T @ PEst[0:S, 0:S] @ G + Cx
```

Comment:
Changed the operator to matrix multiplication instead of elementwise

Reply:
Oh. Thank is a bug. Thanks.